### ISSUE TITLE
Picklist accessibility: focus on Escape; communicate selected state.

### ISSUE THREAD

**Issue by minevskiy** [salesforce/design-system-react#947]
From accessibility review of Lightning Dashboards:
1. inline Picklist doesn't close on Escape. When a Picklist is embedded into a Popover, Escape causes the Popover to close. When Escape is pressed to close the Picklist, user focus should be placed on the trigger button.
2. Picklist doesn’t communicate selected state (using VoiceOver)

Demo code for issue 1 (based on https://react.lightningdesignsystem.com/components/popovers/)
```
import React from 'react';
import Popover from 'design-system-react/components/popover'; 
import Button from 'design-system-react/components/button';
import Icon from '../../components/icon';

const panelContent = (
  <Picklist
     isInline={true}
     label="Show Me"
     options={[
       { label: 'All Products', value: 'all-products' },
       { label: 'All Wackamoles', value: 'all-Wackamoles' }
    ]}
    placeholder="Select record type"
  />
);

const Example = React.createClass({
	displayName: 'PopoverExample',

	render () {
		return (
			<div>
				<Popover
					ariaLabelledby="ALTERNATIVE-HEADING"
					body={panelContent}
				>
					<Button label="Trigger Popover" />
				</Popover>
			</div>
		);
	}
});

ReactDOM.render(<Example />, mountNode);
```

Demo for issue 2: https://react.lightningdesignsystem.com/components/picklist-menus/

@SiTaggart

**Comment by interactivellama** [comment]
Closing. Feel free to re-open for Combobox.
